src/ui/osx/TogglDesktop/test2/AppDelegate.m

```diff
@@ -470,12 +470,15 @@ - (void)new:(TimeEntryViewItem *)new_time_entry
 	NSAssert([NSThread isMainThread], @"Rendering stuff should happen on main thread");
 	NSAssert(new_time_entry != nil, @"new time entry details cannot be nil");
 
+    // Start or create empty TE from Timer mode
+    NSString *duration = self.manualMode ? @"0" : new_time_entry.Description;
```

Comment:
Typo here, should be `duration` not `Description` I fixed it with e7609c69f23d423aacc72c751e48266c8d5e382a